### ISSUE TITLE
[Feature Request] Support GeoJson

### ISSUE THREAD

**Issue by ytmagalhaes** [chancejs/chancejs#424]
We work with `GeoJson` a lot as it's a standard for drawing geometry on maps. It would be nice if we could do `chance.geometry()` and it would give us a random GeoJson.

An options should definitely be the type of geometry (point, linestring, polygon, etc...).

**Comment by matheusfcorocher** [comment]
I was verifying your request, and there is a function called geojson but today only returns a string concatenation of longitude+latitude+eleveation.

@victorquinn, do you think the actual return of geojson should change from string to format describe in this document(https://datatracker.ietf.org/doc/html/rfc7946)?

In this IBM's document is more clear the geojson format: https://www.ibm.com/docs/en/db2/11.5?topic=formats-geojson-format

**Comment by St1fle** [comment]
Hey @matheusfcorocher !

I'm the original author, I just decided to use my personal account for this. Thank you so much for looking at this, it's been 3 years so I never thought anyone was going to be able to add this feature!

You're looking at the right place, the official website for GeoJSON is here https://geojson.org/ and points to https://datatracker.ietf.org/doc/html/rfc7946 which is what most geographical tools use (including on the web).

In case you want to see an example of an implementation of this, there is an npm package that introduces types for GeoJSON, I think the one we are using is this one: https://github.com/caseycesari/geojson.js

But feel free to introduce new types yourself, so long as it follows the spec it should not be a problem.
Also feel free to keep the current implementation of `chance.geometry()`, the one that returns a string concatenation of longitude and lattitude and make a new method like `change.geojson()` if you want, we don't have a problem with the current implementation we just want something that follows the official spec.

Thank you again and hope that helps!

**Comment by matheusfcorocher** [comment]
Hey @St1fle,

Thanks for helping me out with more resources, however we need to await @victorquinn make a response about this feature. I think the easiest way of doing it is to use geojson.js from caseycesari and feed it with latitude, longitude, elevation and other properties. But chance.js doesn't use dependencies, so this is another problem.

**Comment by victorquinn** [comment]
Yeah so I agree this should conform to the geojson spec and it doesn't currently. 

Honestly I'm not sure how that made it in there, it's not documented and it doesn't make much sense for the geojson function to return something that is not geojson. So I'm a big fan of fixing that. 

Also do not want to pull in additional dependencies at this time so would prefer not to pull in that library, but it shouldn't be too terribly hard to implement it without the library and just return something which conforms to the spec.
